src/box/lua/slab.cc

```diff
@@ -271,7 +275,9 @@ lbox_runtime_info(struct lua_State *L)
 static int
 lbox_slab_check(MAYBE_UNUSED struct lua_State *L)
 {
+#ifndef SMALL_MALLOC_IMPL
 	slab_cache_check(SmallAlloc::get_alloc()->cache);
+#endif
```

Comment:
This function should be available in ASAN build. It may be a no-op.

Reply:
Introduced and used `small_check`:
```diff
@@ -271,7 +271,7 @@ lbox_runtime_info(struct lua_State *L)
 static int
 lbox_slab_check(MAYBE_UNUSED struct lua_State *L)
 {
-       slab_cache_check(SmallAlloc::get_alloc()->cache);
+       small_check(SmallAlloc::get_alloc());
        return 0;
 }
```